### DISCUSSION TITLE
Injected Metamask Support?

### DISCUSSION THREAD

**Discussion by BuchananQuantum** [WalletConnect/walletconnect-monorepo#1345] [Developer Support] [answered]
Currently integrating 2.0.0-rc.1 with Ceramic (3Box). 

My question lies with the Metamask mobile app. 
Why am I unable to connect my Metamask mobile app  using v2 but it was feasible with v1.   Are you no longer offering support for Metamask or are you making the assumption most users are leveraging the Metamask browser plugin?

**Comment by chadyj** [comment, marked as answer]
Metamask hasn't yet integrated v2, so it won't work. You will need a v2 compatible wallet for v2 to work.

The community is still migrating to v2, so support is still ongoing.

For dapps, we recommend integrating v1 for the time being.
For wallets, we recommend integrating both v1 and v2.

**Reply by BuchananQuantum** [reply]
Alrighty then!
